### PR TITLE
Doc: uploaded documentation files

### LINKED ISSUE #32: upload documentation files accordingly
Upload  documentation files as follows
- [x] 1. SRS
- [x] 2. PPT
- [ ] 3. Report

### PR DESCRIPTION
**Pull request by patilprthm**
Following documents are uploaded 
1. SRS

Closes #32

### CODE REVIEW

**Comment by malivinayak** [comment]
@OutdatedGuy Please check changed files as multiple files are reflecting change due to rebase with dev

**Review by malivinayak — COMMENTED** [review]
SRS is appropriate

**Review by OutdatedGuy — DISMISSED** [review]
Great 👍🏼

**Review by malivinayak — CHANGES_REQUESTED** [review]
@Pratham1402 Update [README.md](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/README.md) with hyperlink to directly navigate to SRS and PPT's

Note: Update [README.md](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/README.md) which is located inside [doc](https://github.com/malivinayak/Trip-Management-System/tree/doc/doc) folder

File: doc/README.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+So, customers do not need to call & spend unnecessary time in order to reserve their 
+preferred vehicle.
+
+## Documentation
+
+- [Idea Presentation](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/Idea%20Presentation/ADS%20PBL%20SRS%20PPT.pdf)
+
+- [Synopsis]()
+
+- [SRS](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/SRS/Group_06_ADS_PBL_SRS.docx)
```
Comment (malivinayak) [inline doc/README.md]: .docx file is not visible 
Upload .pdf and give a relative path

File: doc/README.md [hunk 2]
```diff
@@ -1 +1,13 @@
-# Document
+# Trip Management System with Advanced Database
+
+‘Trip Management System’ is to automate vehicle rental and reservation. 
+So, customers do not need to call & spend unnecessary time in order to reserve their 
+preferred vehicle.
+
+## Documentation
+
+- [Idea Presentation](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/Idea%20Presentation/ADS%20PBL%20SRS%20PPT.pdf)
```
Comment (malivinayak) [inline doc/README.md]: embedding link will consume time and data
Use relative path this is a efficient way
e.g.
```
- [Idea Presentation](./Idea%20Presentation/ADS%20PBL%20SRS%20PPT.pdf)
```

File: doc/README.md [hunk 3]
```diff
@@ -1 +1,13 @@
-# Document
+# Trip Management System with Advanced Database
+
+‘Trip Management System’ is to automate vehicle rental and reservation. 
+So, customers do not need to call & spend unnecessary time in order to reserve their 
+preferred vehicle.
+
+## Documentation
```
Comment (malivinayak) [inline doc/README.md]: can you take any other heading here

**Review by malivinayak — CHANGES_REQUESTED** [review]

**Comment by malivinayak** [comment]
@Pratham1402 I have requested few changes 
Take a look

File: doc/README.md (hunk 1 above)
Comment (patilprthm) [reply doc/README.md]: Done!

File: doc/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+- [Idea Presentation](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/Idea%20Presentation/ADS%20PBL%20SRS%20PPT.pdf)
+
+- [Synopsis]()
+
+- [SRS](https://github.com/malivinayak/Trip-Management-System/blob/doc/doc/SRS/Group_06_ADS_PBL_SRS.pdf)
```
Comment (patilprthm) [inline doc/README.md]: Done!

File: doc/README.md (hunk 2 above)
Comment (patilprthm) [reply doc/README.md]: Done!

File: doc/README.md (hunk 3 above)
Comment (patilprthm) [reply doc/README.md]: Done!

**Comment by patilprthm** [comment]
> @Pratham1402 I have requested few changes Take a look

**Comment by patilprthm** [comment]
> > @Pratham1402 I have requested few changes Take a look

Updated the changes. Take a look on it!

**Review by malivinayak — APPROVED** [review]

**Pull request merged**